mixer/mixer.ino

```diff
@@ -64,7 +64,7 @@ ESP8266WebServer server(80);
 
 
 #include <Wire.h>
-#include "Adafruit_MCP23017.h"
+#include "src/Adafruit_MCP23017_Arduino_Library/Adafruit_MCP23017.h"
```

Comment:
Let's simplify the path and make it shorter
like so
`#include "src/Adafruit_MCP23017/Adafruit_MCP23017.h"`